data/tlg0065/tlg001/tlg0065.tlg001.1st1K-grc1.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-    <teiHeader>
+    <teiHeader xml:lang="eng">
         <fileDesc>
             <titleStmt>
                 <title xml:lang="lat">Periplus Scylacis</title>
                 <author>Scylax</author>
-                <editor xml:lang="lat">CAROLUS MULLERUS</editor>
-                <sponsor>HARVARD COLLEGE LIBRARY</sponsor>
+                <editor xml:lang="lat">Carolus Müllerus</editor>
+                <sponsor>Harvard College Library</sponsor>
```

Comment:
See note on data/tlg0064/tlg001/tlg0064.tlg001.1st1K-grc1.xml for the correct sequence of funders and `<respStmt>` elements.